### ISSUE TITLE
masks = torch.cat([masks]*3, dim = 1) 

### ISSUE THREAD

**Issue by jsjliuyun** [jingyuanli001/RFR-Inpainting#23]

**Comment by jsjliuyun** [comment]
I have resolved this question.

**Comment by sriderya** [comment]
Hi,

This line of code gives error because it creates an input with 9 channels. What is your solution about this line? Commenting it out may be a solution but I am not sure if it is a correct way and I don't know how it will affect the rest of the model.

Thanks
